src/ipfs-core/init.js

```diff
@@ -0,0 +1,98 @@
+const peerId = require('peer-id')
+
+module.exports = (repo, opts, callback) => {
+  opts = opts || {}
+  opts.force = opts.force || false
+  opts.emptyRepo = opts.emptyRepo || false
+  opts.bits = opts.bits || 2048
+
+  // Pre-set config values.
+  var config = {
```

Comment:
Would be nicer if this was a separate json file, so it's easier/clearer when things change.